xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-api/src/test/java/org/xwiki/refactoring/script/RequestFactoryTest.java

```diff
@@ -135,6 +142,10 @@ public void createMoveRequest() throws Exception
         assertTrue(moveRequest.isAutoRedirect());
         assertFalse(moveRequest.isInteractive());
         assertTrue(moveRequest.isCheckRights());
+
+        // check added for fixing pseudo-tested method
```

Comment:
I don't like too much this comment because it feels like we're adding this only for Descartes while we should only add tests because it adds value.

Reply:
so you propose I just delete the comment?

Reply:
I guess, yes, since what it does seems obvious (even if not very interesting...)